examples/ruby/devenv.nix

```diff
@@ -3,7 +3,21 @@
 {
   languages.ruby.enable = true;
 
+  # turn off C tooling if you do not intend to compile native extensions, enabled by default
+  # languages.c.enable = false;
+
   enterShell = ''
+    echo 'Making sure the basics for native compilation are available:'
```

Comment:
These will fail if C language is disabled, so let's just remove the prints :)

Reply:
I think this is the side-effect of the current examples based testing approach.

I view this 'enterShell' block as a test assertion on the configuration.

I think you mean to keep the examples as minimal as possible, which is fair. It would be great to have both use-cases covered.

Maybe a devenv.local.nix could contain the more assertiony configuration? How does that sound?

Reply:
I need to think about this, eventually I'd like each example have README.md that's tested by https://github.com/OceanSprint/tesh